src/main/java/Duke.java

```diff
@@ -135,44 +159,46 @@ private static void addTask(Task t) {
     private static String executeAddTodo(String description) {
         addTask(new Todo(description));
         todosCount++;
-        return "added: " + tasks[tasksCount - 1].toString() + LS + String.format(MESSAGE_TASKS_FOUND_OVERVIEW, todosCount, COMMAND_TODO_WORD);
+        return MESSAGE_ADDED + tasks[tasksCount - 1].toString() + NEW_LINE
+                + String.format(MESSAGE_TASKS_FOUND_OVERVIEW, todosCount, COMMAND_TODO_WORD);
     }
 
     private static String executeAddEvent(String commandArgs) {
         String[] eventAndDate = decodeTask(COMMAND_EVENT_WORD, commandArgs);
```

Comment:
I think according to the coding standard we learnt, the name for the string array should be in plural form, so I guess this should be rename as eventAndDates :)